spec/services/articles/creator_spec.rb

```diff
@@ -12,9 +12,10 @@
       end.to change(Article, :count).by(1)
     end
 
-    it "returns an article" do
+    it "returns a non decorated, persisted, article" do
```

Comment:
```suggestion
    it "returns a non decorated, persisted article" do
```

Reply:
as you've already approved and this is not a blocker I'm going to wait merging this, as it'll invalidate all the reviews. I'll write it down for a following PR

Reply:
Sounds good! Definitely don't need to invalidate reviews for this